en/what-do-i-need-to-publish/list-of-all-elements

```diff
@@ -0,0 +1,38 @@
+Full list of top-level elements in IATI
+=========
+
+| Topic  | Element | Mandatory | Recommended (minimum) |
+| --- | --- | :---: | :---: |
+| Identification  | IATI Identifier  | Yes |
+|  | Reporting Org | Yes |
+|  | Other Identifier |
+| Basic Activity Information | Title | Yes |
+|  | Description | Yes |
+|  | Activity Status | Yes | 
+|  | Activity Date | Yes |
+|  | Contact Info |
+|  | Activity Scope |
+| Participating Organisations | Participating Org | Yes |
+| Geopolitical Information | Recipient Country |  | Yes |
+|  | Or, Recipient Region |  | Yes |
+|  | Location |
```

Comment:
There are a number of rows with nothing in either the `Mandatory` or `Recommended (minimum)` column. This seems a bit confusing.

Reply:
It follows from the previous page 'what do I need to publish'. But this time with blank rows of all the non 'minimum' elements.